File: Java/LogicaProgramacaoJava(Eclipse)/exerciciosJava/src/manzanoPag50/ExercicioG.java
package manzanoPag50;

/*Elaborar um programa que apresente como resultado o valor do fatorial dos valores ímpares 
situados na faixa numérica de 1 a 10*/

public class ExercicioG {

	public static void main(String[] args) {

		int contadora = 0;
		int antecessor = 0; /* inteiro */
		int resultado = 0;
		int fatorial = 0;
		float guardaValor = 0; /* real */

		do {
			contadora = contadora + 1;
			antecessor = contadora;
			fatorial = contadora;

			if ((contadora % 2) == 1) {
				System.out.println("O valor " + contadora + " é impar");

				do {
					antecessor = antecessor - 1;
					fatorial = fatorial * antecessor;
					resultado = fatorial;
					System.out.println(resultado);
				} while (antecessor > 2);

				guardaValor = guardaValor + resultado;
			}

		} while (contadora < 10);

		System.out.println("A soma do fatorial de todos os numeros digitado é " + guardaValor);

	}

}

File: ExerciciosJavaScript(LógicadeProgramacao)/exercicios pag 50(Manzano)/js/exercicioG.js
/*Elaborar um programa que apresente como resultado o valor do fatorial dos valores ímpares 
situados na faixa numérica de 1 a 10*/

var contadora = 0
var antecessor = 0    /*inteiro*/
var resultado = 0  
var fatorial = 0  
var guardaValor = 0 /*real*/

do {
    contadora = contadora + 1
    antecessor = contadora
    fatorial = contadora

    if ((contadora % 2) == 1) {
        alert("O valor " + contadora + " é impar")

        do {
            antecessor = antecessor - 1
            fatorial = fatorial * antecessor
            resultado = fatorial
            alert(resultado)
        } while (antecessor > 2)

        guardaValor = guardaValor + resultado
    }

} while (contadora < 10)

alert("A soma do fatorial de todos os numeros digitado é " + guardaValor)